tests/test_functional.py

```diff
@@ -0,0 +1,92 @@
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import pytest
+import obal
+
+
+FIXTURE_DIR = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'fixtures',
+)
+
+TESTREPO_DIR = os.path.join(FIXTURE_DIR, 'testrepo')
+MOCKBIN_DIR = os.path.join(FIXTURE_DIR, 'mockbin')
+
+DEFAULT_ARGS = []
+if os.environ.get('TRAVIS', None):
+    DEFAULT_ARGS.extend(['-e', 'ansible_remote_tmp=/tmp/ansible-remote'])
+
+
+def obal_cli_test(func):
+    def func_wrapper(*args, **kwargs):
+        tempdir = tempfile.mkdtemp()
```

Comment:
pytest has a [tmpdir fixture](https://docs.pytest.org/en/latest/tmpdir.html). Would that work here as well?